### PR TITLE
ADD account_invoice_company_search , journal_filter

### PR DESCRIPTION
**Pull request by nicomacr**

### CODE REVIEW

**Comment by guewen** [comment]
Hi,

Thanks for your proposal.

Travis reported lint errors:

```
flake8 . --config=${FLAKE8_CONFIG_DIR}/travis_run_flake8__init__.cfg
./account_invoice_company_search/__init__.py:4:79: W292 no newline at end of file
./account_invoice_journal_filter/__init__.py:3:1: W391 blank line at end of file
status1=$?
flake8 . --config=${FLAKE8_CONFIG_DIR}/travis_run_flake8.cfg
./account_invoice_company_search/__openerp__.py:35:5: E122 continuation line missing indentation or outdented
./account_invoice_journal_filter/__openerp__.py:49:66: W292 no newline at end of file
```

Could you check that?

Regarding the names of your modules, the first one ends with `_search` and the second one with `_filter` but they have the same goal (on different fields). Could you choose one of the suffixes?

File: account_invoice_company_search/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{
+    "name": "Account Invioce Company Search",
+    "version": "1.0",
+    'author':  'ADHOC',
+    'website': 'www.adhoc.com.ar',
+    "category": "Accounting",
+    "description": """
+Account Invioce Company Search
+==============================
+Adds company search filter in account invoice view
```
Comment (guewen) [inline account_invoice_company_search/__openerp__.py:29]: To be well displayed on apps.odoo.com, starting from the version 8.0, we remove the description key  from `__openerp__.py` and instead we add a README.rst  file based on this template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: account_invoice_company_search/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Account Invioce Company Search",
```
Comment (guewen) [inline account_invoice_company_search/__openerp__.py:21]: s/Invioce/Invoice/

File: account_invoice_company_search/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "version": "1.0",
+    'author':  'ADHOC',
```
Comment (guewen) [inline account_invoice_company_search/__openerp__.py:23]: Could you put `ADHOC,Odoo Community Association (OCA)` in the author (it is used on apps.odoo.com to show all the OCA's modules)?

File: account_invoice_journal_filter/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+{
+    "name": "Account Invoice Journal Filter",
+    'version': '1.0',
+    'category': 'Accounting',
+    'sequence': 14,
+    'summary': 'Quotations, Sales Orders, Invoicing',
+    'description': """
+Account Invoice Journal Filter
+====================
+Agrupar facturas por diarios
```
Comment (guewen) [inline account_invoice_journal_filter/__openerp__.py:29]: Sames remarks than above + translation

**Comment by legalsylvain** [comment]
@nicomacr : What about the remarks made by @guewen ? Thanks.

**Pull request closed without merging**